Automattic/wp-calypso
# Domains: Add enable/disable privacy for registered domains

#### Changes proposed in this Pull Request

We made privacy free, so now we want to simplify the logic
and allow users to just be able to toggle privacy instead of
having to go through checkout, or to cancel the privacy sub
every time.

Depends on D24443-code.

#### Testing instructions

- Open a registered domain
- Go to privacy screen
- Toggle privacy on-off
- Make sure all is working correctly
- Confirm the removed screen is used nowhere else
